tests/network/sriov.go

```diff
@@ -620,8 +620,8 @@ func defaultCloudInitNetworkData() string {
 
 func findIfaceByMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, mac string, timeout time.Duration) (string, error) {
 	var ifaceName string
-	err := wait.Poll(timeout, 5*time.Second, func() (done bool, err error) {
-		vmi, err := virtClient.VirtualMachineInstance(vmi.GetNamespace()).Get(context.Background(), vmi.GetName(), k8smetav1.GetOptions{})
+	err := wait.PollUntilContextTimeout(context.Background(), timeout, 5*time.Second, true, func(ctx context.Context) (done bool, err error) {
```

Comment:
Usage of a boolean directly is not advised as it is hard to read.
Please use a const.